static/.htaccess

```diff
@@ -8,7 +8,7 @@ Options -Indexes -MultiViews
 
 # Redirect pages - https://httpd.apache.org/docs/current/mod/mod_rewrite.html#rewriterule
 
-# deduplication of Web Modeler API
+<!-- # deduplication of Web Modeler API
```

Comment:
I don't think these HTML-style comments actually work in an htaccess, but that's what VS Code did for me when I tried to comment out an entire block.

The point of these comments is that in the real implementation, I'd remove *all* of these redirects that don't pertain to 1.3. I commented them out here, instead of removing them, because removing would have been super noisy.